### DISCUSSION TITLE
v0.8

### DISCUSSION THREAD

**Discussion by alphapapa** [alphapapa/org-ql#386] [Announcements]
**Additions**

-   Function `org-ql-completing-read`, used by command `org-ql-find`, now specifies the completion category as `org-heading`, providing compatibility with [Embark](https://github.com/oantolin/embark).  (This is a powerful feature, as it means any `org-ql-find` result can be acted on from inside the search results with Embark, which provides common actions from Org Agenda and Org speed keys bindings.)  ([#299](https://github.com/alphapapa/org-ql/issues/299).  Thanks to [Omar Antolín Camarena](https://github.com/oantolin), [Daniel Mendler](https://github.com/minad), and [Akira Komamura](https://github.com/akirak).)
    -   Command `org-ql-completing-read-export`, bound to `C-c C-e` or `embark-export` while in an `org-ql-completing-read` session, exits and shows an `org-ql-view` buffer for the current search.
-   Command `org-ql-find` may be called in an `org-agenda` or `org-ql-view` buffer to search the buffers which contributed to the agenda/view buffer.
-   Command `org-ql-find-path`, which searches outline paths in the current buffer.
-   Command `org-ql-open-link`, which finds links in entries matching the given query, and opens the selected one with `org-open-at-point`.  (This is helpful when a collection of links are kept in Org files: rather than having to first visit the entry containing the desired link, then locate it within the entry, and then open it, the user can simply select the link and open it directly.)
-   Items in `org-ql-view` buffers now include the `org-category` text property, like Org Agenda buffers, which allows grouping with `org-super-agenda`'s category-related selectors.  ([#363](https://github.com/alphapapa/org-ql/issues/363).  Thanks to [Gabriele Mongiano](https://github.com/kofm) for reporting.)

**Fixes**

-   Predicate `property` correctly uses the value of `org-use-property-inheritance` when not specified.  ([#346](https://github.com/alphapapa/org-ql/pull/346), [#356](https://github.com/alphapapa/org-ql/issues/356).  Thanks to [Bram Schoenmakers](https://github.com/bram85).)

**Compatibility**

-   Emacs 27.1 or later is now required.
-   Org v9.7's `org-element` API changes required some adjustments.  ([#364](https://github.com/alphapapa/org-ql/issues/364).  Thanks to several users for reporting, and to [Ihor Radchenko](https://github.com/yantar92) for his feedback.)

<hr /><em>This discussion was created from the release <a href='https://github.com/alphapapa/org-ql/releases/tag/v0.8'>v0.8</a>.</em>

**Comment by ivanperez-keera** [comment]
Thank you for the release, Adam. I was no longer able to do my regular planning with the slow and inexpressive sparse tree search mechanisms provided by Emacs org-mode. I was really considering to switch tools until someone pointed me to `org-ql`.

`org-ql` has been an absolute game changer.

It's great to see releases being published!

**Reply by alphapapa** [reply]
Hi Ivan,

Thank you for the kind words.  I'm glad to hear that it's useful to you.  If you ever get the chance, I'd be glad to read more about how you use Org and org-ql to do your work, maybe in a blog post or something.  And please do send any feedback you may have, as it sounds like you are one of org-ql's more serious users.  I still have a number of major features I want to add to org-ql before I consider it "1.0", and I'm always looking for new ideas.
